src/pretix/base/validators.py

```diff
@@ -0,0 +1,45 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+class BlacklistValidator:
+
+    blacklist = []
+
+    def __call__(self, value):
+        # Validation logic
+        if value in self.blacklist:
+            message = _("This slug has an invalid value.")
```

Comment:
I think having the failing slug in the message would be even better (appended, for example).